<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualGraphDialog</class>
 <widget class="QDialog" name="VisualGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_x_axis">
           <property name="text">
            <string>X Axis: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_x_axis"/>
         </item>
         <item>
          <widget class="QLabel" name="label_y_axis">
           <property name="text">
            <string>Y Axis: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_y_axis"/>
         </item>
         <item>
          <widget class="QLabel" name="label_z_axis">
           <property name="text">
            <string>Z Axis: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_z_axis"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="fill"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
